Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int[] arr = new int[]{1, 2, 3, 4, 5};
+        System.out.println("Even numbers:");
+
+        for (int i = 0; i < arr.length; i = i * 2)

```

Comment:
**Scalability Issue**: The loop increment 'i = i * 2' can lead to an infinite loop for arrays of size greater than 1, severely impacting scalability by causing unnecessary CPU usage and potentially crashing the system under high load. <br> **Fix**: Change the loop increment to a more predictable and terminating condition, such as 'i += 2' for iterating over even indices, ensuring the loop terminates correctly. <br> **Code Suggestion**: 
 ```
 for (int i = 0; i < arr.length; i += 2) {
     System.out.println(arr[i]);
 }
 ```